packages/dx-react-core/docs/guides/fundamentals.md

```diff
@@ -0,0 +1,41 @@
+# React Core - Fundamentals
+
+React Core provides a set of React components that allows to implement pluggable React component.
+
+## Pluggable React Component
+
+Pluggable React component is a component that transforms special React components (plugins) into the complex React component hierarchy. It also provides an ability to communicate between plugins.
+
+Pluggable React component aggregates two core principles:
+
+- **[Inversion of Control](https://en.wikipedia.org/wiki/Inversion_of_control)**. It transforms linearly component structure defined by plugins into rich React component tree. It allows to define points in markup that can be extended by other plugins.
+- **[State Managment](https://en.wikipedia.org/wiki/State_management)**. It provides a basic communication between plugins. It allows to define values and actions that can be consumed and executed by other plugins.
```

Comment:
If it possible to open this links in a new tab, I believe that we need to do this, because this links way out our site.

Reply:
+1 for all external links to be opened in a new tab